optiga/include/optiga/optiga_lib_config_m_v3.h

```diff
@@ -110,7 +110,7 @@ extern "C" {
      *         Warm Reset - (2) : This is applicable if the host platform doesn't have GPIO option for VDD. \n
      *         Any other value will lead to error
      */
-    #define OPTIGA_COMMS_DEFAULT_RESET_TYPE     (0U)
+    #define OPTIGA_COMMS_DEFAULT_RESET_TYPE     (1U)

```

Comment:
this should be 0 by default, what we can do change it to the #ifndef OPTIGA_COMMS_DEFAULT_RESET_TYPE #define ...